version_template.py

```diff
@@ -55,7 +55,7 @@ def exactPython():
     for. Something like 'python2.7' or 'python3.6'.
     """
     import sys
-    return 'python{}.{}'.format(sys.version_info[0], sys.version_info[1])
+    return f'python{sys.version_info[0]}.{sys.version_info[1]}'
```

Comment:
Is this really clearer this way?

Reply:
It is more compact; most IDEs will make this fairly easy to read